ZUGFeRD/InvoiceDescriptor21Reader.cs

```diff
@@ -387,6 +396,37 @@ private static TradeLineItem _parseTradeLineItem(XmlNode tradeLineItem, XmlNames
                 };
             }
 
+            if (tradeLineItem.SelectSingleNode(".//ram:SpecifiedLineTradeSettlement", nsmgr) != null)
+            {
+                XmlNodeList LineTradeSettlementNodes = tradeLineItem.SelectSingleNode(".//ram:SpecifiedLineTradeSettlement", nsmgr).ChildNodes;
+                foreach (XmlNode LineTradeSettlementNode in LineTradeSettlementNodes)
+                {
+                    switch (LineTradeSettlementNode.Name)
+                    {
+                        case "ram:ApplicableTradeTax":
+                            //TODO
+                            break;
+                        case "ram:BillingSpecifiedPeriod":
+                            //TODO
+                            break;
+                        case "ram:SpecifiedTradeAllowanceCharge":
+                            //TODO
+                            break;
+                        case "ram:SpecifiedTradeSettlementLineMonetarySummation":
+                            //TODO
+                            break;
+                        case "ram:AdditionalReferencedDocument":
+                            //TODO
+                            break;
+                        case "ram:ReceivableSpecifiedTradeAccountingAccount":
+                            ReceivableSpecifiedTradeAccountingAccount RSTAA = new ReceivableSpecifiedTradeAccountingAccount();
```

Comment:
could you use the object initializer approach here, directly initializing all properties in one go (just like you did in all other changes)?

Reply:
You are right, I also changed these lines.